pyhf/commandline.py

```diff
@@ -105,11 +104,7 @@ def cls(workspace, output_file, measurement, qualify_names, patch):
             with click.open_file(p, 'r') as read_file:
                 p = jsonpatch.JsonPatch(json.loads(read_file.read()))
             spec = p.apply(spec)
-        p = Model(
-            spec,
-            poiname=measurements[measurement_index]['config']['poi'],
-            qualify_names=qualify_names,
```

Comment:
this keyword arg in Model will become superfluous and should thus be removed (+ any logic using it)